test/src/Makefile

```diff
@@ -14,13 +14,16 @@ init:
 .PHONY : test
 ## Run tests
 test: init
-	# go mod download
-	go test -v -timeout 60m -run TestExamplesComplete
+	# Verify we have valid AWS credential
+	if command -v aws >/dev/null; then \
+	  (unset AWS_PROFILE; aws sts get-caller-identity >/dev/null); \
+	fi
+	go test -v -timeout 10m -run TestExamplesComplete
```

Comment:
@Nuru I'm not 100% sure entirely why this is, but since `cloudposse/actions` sets `AWS_SECRET_KEY` instead of the newer `AWS_SECRET_ACCESS_KEY` https://github.com/cloudposse/actions/blob/tf-v1/.github/workflows/test-command.yml#L403 `aws sts get-caller-identity` will fail, even though `terratest` will not. I think this is because the AWS golang SDK maintains backwards compatibility between `AWS_SECRET_ACCESS_KEY` and `AWS_SECRET_KEY`, but the cli does not.

So, `aws sts get-caller-identity` before invoking `terratest` will cause the run to fail

https://github.com/cloudposse/actions/runs/2925169210?check_suite_focus=true

```
Run make -C test/src
  make -C test/src
  shell: bash -e -o pipefail {0}
  env:
    MAKE_INCLUDES: Makefile
    AWS_ACCESS_KEY_ID: ***
    AWS_SECRET_KEY: ***
make: Entering directory '/__w/actions/actions/test/src'
# Verify we have valid AWS credential
if command -v aws >/dev/null; then \
  (unset AWS_PROFILE; aws sts get-caller-identity >/dev/null); \
fi

Partial credentials found in env, missing: AWS_SECRET_ACCESS_KEY
make: *** [Makefile:18: test] Error 255
make: Leaving directory '/__w/actions/actions/test/src'
Error: Process completed with exit code 2.
```